frontend/Container.js

```diff
@@ -133,6 +133,11 @@ var DEFAULT_MENU_ITEMS = {
         title: 'Show all ' + node.get('name'),
         action: () => store.changeSearch(node.get('name')),
       });
+      items.push({
+        key: 'copyNodeName',
+        title: 'Copy element name',
+        action: () => store.copyNodeName(node.get('name')),
+      });
```

Comment:
nit: I think this option should be at the bottom (after "Scroll to node")

I was also wondering if we should only show it for composite component types (eg `node.get('nodeType') === 'Composite'`)? It seems a bit unlikely that someone would want to copy "div" or "span" although I guess it doesn't hurt to have the menu option there either, if it makes things simpler.